registry/storage/driver/walk_test.go

```diff
@@ -45,3 +72,217 @@ func TestWalkFileRemoved(t *testing.T) {
 		t.Fatalf(err.Error())
 	}
 }
+
```

Comment:
In retrospect, I probably could have made this table drive, but oh well 🤷🏻‍♂️ lmk if this is worth changing ?